### PR TITLE
Blog - Intro to Shipwright pt 1

### PR DESCRIPTION
**Pull request by adambkaplan**
Blog post introducing project Shipwright. This will be done in two parts,
the first setting the scene with a history of building applications. The
experiences here are based on my own - first building JARs at a small
software shop, then deploying images on k8s at a startup.

### CODE REVIEW

**Review by otaviof — APPROVED** [review]

File: content/en/blog/posts/2020-10-15-intro-shipwright-pt1/index.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+## Delivering on Docker and Kubernetes
+
+The emergence of Docker and Kubernetes changed the unit of delivery. Teams no longer delivered
+executables, JAR files, or script bundles when releasing software. To run an application in Docker
+or Kubernetes environments, teams needed to package their software into a container image. This was
+not a simple task, since many teams now had to learn entirely new sets of tools and skills to deploy
+their code.
+
+I first learned of Docker and Kubernetes at a startup I had joined. We used Kubernetes as a means
+to scale our back-end application and break apart our monolith build with Python and Django. To
```
Comment (coreydaley-redhat) [inline content/en/blog/posts/2020-10-15-intro-shipwright-pt1/index.md:47]: s/build/built

File: content/en/blog/posts/2020-10-15-intro-shipwright-pt1/index.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+a change control review with our client's IT department, our software would be released during a
+scheduled maintenance window.
+
+For engineers in larger enterprises, this experience should feel familiar. You may have used C#,
+C++, or were adventurous and testing Ruby on Rails. Perhaps instead of compiling the application
+yourself, a separate release team was responsible for building the application on secured
+infrastructure. Your releases may have undergone extensive acceptance testing in a staging
+environment before being promoted to production (and those practices may still continue today). If
+you were fortunate, some of the release tasks were automated by emerging continuous integration
+tools - Hudson and Jenkins.
```
Comment (coreydaley-redhat) [inline content/en/blog/posts/2020-10-15-intro-shipwright-pt1/index.md:36]: s/-/such as/

File: content/en/blog/posts/2020-10-15-intro-shipwright-pt1/index.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+executables, JAR files, or script bundles when releasing software. To run an application in Docker
+or Kubernetes environments, teams needed to package their software into a container image. This was
+not a simple task, since many teams now had to learn entirely new sets of tools and skills to deploy
+their code.
+
+I first learned of Docker and Kubernetes at a startup I had joined. We used Kubernetes as a means
+to scale our back-end application and break apart our monolith build with Python and Django. To
+test our applications, we built our container images locally with Docker, and ran clusters locally
+with  minikube or used a dev cluster set up with our cloud provider. For acceptance testing and
+production releases, we used a third-party continuous integration service to assemble to our code
```
Comment (coreydaley-redhat) [inline content/en/blog/posts/2020-10-15-intro-shipwright-pt1/index.md:50]: s/assemble to our code/assemble our code/

**Review by coreydaley-redhat — COMMENTED** [review]
Just a few grammatical nits

**Review by qu1queee — APPROVED** [review]
Thanks Adam, part 1 looks like a nice introduction for readers for being able to answer, "why do I need the Build API?"

**Comment by adambkaplan** [comment]
@coreydaley-redhat @qu1queee removing WIP. Updated grammar, added images and links.

**Pull request merged**